specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/providers.json

```diff
@@ -0,0 +1,288 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.KeyVault/operations": {
+      "get": {
```

Comment:
I'm not sure why you got this given you have the providers.json file here... did you not reference the new version in your readme?

"code: InvalidResourceType,
message: The resource type 'operations' could not be found in the namespace 'Microsoft.KeyVault' for api version '2022-11-01'. The supported api-versions are '2014-12-19-preview,

Reply:
Maybe there is something wrong with the path here. Is it supposed to be a subscription scope?

Reply:
This seems to be because the service is not deployed. Once this PR is signed off then I will start service deployment.